### ISSUE TITLE
Support federation of moderation notes

### ISSUE THREAD

**Issue by ThisIsMissEm** [mastodon/mastodon#27562]
### Pitch

In 2018, we added the ability to have moderation notes on both reports and accounts, allowing moderation teams to better document actions taken and research into reports. In 2023, a major issue facing moderators is how to communicate outside of their instance with other instance's moderation teams.

Whilst adding a generic back-channel for messaging between instance moderation teams is likely not a wise idea (due to its unfocused nature and possibility for abuse by malicious instances), we should definitely look into adding federation of moderation notes on reports (or discussion on reports, as mentioned in https://github.com/mastodon/mastodon/issues/26201). This would allow two or more moderation teams to coordinate a response to a report.

This would need a FEP to be written to define the mechanism for federating these notes, as we'd want to ensure that there is a particular audience for these. Additionally, we may wish to add a ModerationNote object type, such that it would be less likely for these to be accidentally translated into regular Note activities.

The federation of moderation notes shouldn't be automatic, but instead a model of "reply internally" / "reply", much like that which exists in [existing helpdesk / support ticketing software](https://support.zendesk.com/hc/en-us/articles/4408822560410-Changing-the-default-privacy-of-ticket-comments-), such that an instances' moderators can still discuss between each other.

When federated, the moderation notes should only appear as being from the instance actor, such that the individual moderators' privacy is protected (though perhaps there should be a mechanism through which to denote different moderators replying, without revealing their identities)

### Motivation

At present, instance moderators and administrators have few available pathways to communicate between each other, which often results in the Instance Administrator being the "contact point", regardless of whether or not that purpose is actually involved in instance moderation, whilst some instances do specify a contact email, this is only present in the "about this instance" text, and not part of structured data. This provides friction and miscommunication opportunities, often creating unnecessary conflict and claims of an instance being "poorly moderated" because contact attempts fail to be responded to in a "timely" manner (where timely is very debatable as to its definition).

As the fediverse grows, being able to discuss issues with other moderators on other instances will become a growing need.

**Comment by ThisIsMissEm** [comment]
(Just a small note that I've previously mentioned this on discord, but only just now had the time to write up the ticket for it)

**Comment by BakaDalek** [comment]
I support this if all existing moderation notes default to internal only and moving forward it is very clearly labeled that the moderation note is being posted publicly or internally 👍

**Comment by ThisIsMissEm** [comment]
Yes, all existing notes would remain internal. There'd also need to be some design work done here, but I think it'd be possible.

Question I have is, if we make them all federate as the instance actor, does that get confusing when >1 person is engaged in the discussion, as from the remote side you'd just see one user: the instance actor, you'd loose the context of "who"

**Comment by ThisIsMissEm** [comment]
The required ActivityPub changes are being tackled by the ActivityPub Trust & Safety Taskforce: https://github.com/swicg/activitypub-trust-and-safety/issues/6
